src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/AuthProviderSpec.scala

```diff
@@ -119,10 +119,6 @@ class AuthProviderSpec extends FreeSpec with ScalatestRouteTest with Matchers wi
       clusterGetResponseException shouldBe a [ClusterNotFoundException]
       clusterGetResponseException.asInstanceOf[ClusterNotFoundException].statusCode shouldBe StatusCodes.NotFound
 
-      //list all clusters should be fine, but empty
```

Comment:
This section of the test will not pass, as all clusters created by a user will be returned in a list clusters call.

Reply:
Is there a way to actually test the new behavior?